collector/fixtures/e2e-output.txt

```diff
@@ -785,43 +788,82 @@ node_md_blocks_synced{device="md10"} 3.14159265e+08
 node_md_blocks_synced{device="md11"} 4.190208e+06
 node_md_blocks_synced{device="md12"} 3.886394368e+09
 node_md_blocks_synced{device="md127"} 3.12319552e+08
+node_md_blocks_synced{device="md13"} 4.88383936e+08
+node_md_blocks_synced{device="md20"} 9.76751616e+08
 node_md_blocks_synced{device="md219"} 7932
 node_md_blocks_synced{device="md3"} 5.853468288e+09
 node_md_blocks_synced{device="md4"} 4.883648e+06
+node_md_blocks_synced{device="md50"} 8.790400512e+09
 node_md_blocks_synced{device="md6"} 1.6775552e+07
 node_md_blocks_synced{device="md7"} 7.813735424e+09
 node_md_blocks_synced{device="md8"} 1.6775552e+07
 node_md_blocks_synced{device="md9"} 523968
-# HELP node_md_disks Total number of disks of device.
+# HELP node_md_disks Number of disks of device. (state="any" equals to the number of disks in the array)
```

Comment:
This is not as easy as it sounds.
"Total number of disks of device" used to refer to the desired number of working disks in the raid.
1. But there could be more disks than that number, i.e. the spare disks.
So the sum would be bigger than total in that case.

2. The number of missing disks is not reported explicitly and sometimes not at all.

As much as I'd like to go along with the pattern, I'm stuck with the limitations of the /proc/mdstat interface.